### PR TITLE
T-shirt preorder form

### PR DESCRIPTION
**Pull request by elliottsj**
Added a T-shirt preorder form and made the drinks/snacks table full-width.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,4 +1,6 @@
 source 'http://rubygems.org'
 
-require 'rbconfig'
-gem 'wdm', '~> 0.1.0' if RbConfig::CONFIG['target_os'] =~ /mswin|mingw/i
```
Comment (arkon) [inline Gemfile:7]: no pls

File: store.html [hunk 2]
```diff
@@ -6,36 +6,41 @@
...
-    {% for snack in site.data.store-prices.snacks %}
-    <tr>
-      <td>{{ snack.name }}</td>
-      <td>{{ snack.price }}</td>
-    </tr>
-    {% endfor %}
-  </tbody>
-</table>
+<section class="store-right">
+  <iframe class="shirt-form" src="https://docs.google.com/forms/d/1ATTDyAhF9jW_sXSfUZkoblGhj4jqWYxWkPOO1u3Re3U/viewform?embedded=true" width="500" height="750" frameborder="0" marginheight="0" marginwidth="0">Loading...</iframe>
```
Comment (arkon) [inline store.html]: Can probably move the width/height/border stuff into the CSS declaration for `.shirt-form`.

Comment (elliottsj) [inline store.html]: Done!

File: Gemfile (hunk 1 above)
Comment (elliottsj) [inline Gemfile:7]: Re-added.

File: store.html (hunk 2 above)
Comment (arkon) [inline store.html]: :thumbsup:

**Comment by jwebbed** [comment]
I want to get pictures up of the shirts before we merge this, also some way to select what shirts you want to buy in the form.

**Comment by arkon** [comment]
Considering I've only finished 1 shirt, I'm guessing this should wait?

**Comment by jwebbed** [comment]
Yes, I also want to merge this one directly into master in case we have it ready before school starts.

**Pull request merged**